tools/cldr-apps/js/src/esm/cldrDeferHelp.mjs

```diff
@@ -45,6 +48,16 @@ function addDeferredHelpTo(fragment, helpHtml, resource, translationHint) {
   $(fragment).append(theHelp);
 }
 
+function linkifyHttpsUrls(text) {
+  const regex = /\b(https:\/\/\S+)/g;
+  const s = text.replace(regex, "<a target='_blank' href='$1'>$1</a>");
+  // In principle, a URL can end in a period;
+  // in practice, a final period in PathDescription.java is not part of the URL
+  const periodQuoteEndBracketRegex = /\.'>/g;
+  const quoteEndBracket = "'>";
+  return s.replace(periodQuoteEndBracketRegex, quoteEndBracket);
+}
+
```

Comment:
lots of problems here.  OK for now, pending https://unicode-org.atlassian.net/browse/CLDR-15708